### DISCUSSION TITLE
User Roles

### DISCUSSION THREAD

**Discussion by ItzBooker** [stoatchat/discussions#834] [Feature Suggestions]
**Please add a display of roles in the user profile. This is a real necessity, thank you.**

**Comment by BrtCgn** [comment]
This is really necessary, I hope they'll do something about it.

**Comment by lezetho** [comment]
I 100% agree. This is one of the biggest necessities that Revolt needs.

**Comment by hiexqwe** [comment]
We can also add roles from users' profiles.

**Reply by lezetho** [reply]
... that's what the post is about
